### PR TITLE
pull to master

### PR DESCRIPTION
**Pull request by kbilsback**

### CODE REVIEW

**Comment by kbilsback** [comment]
try that

File: r_markdown/background_analysis_ecoc.Rmd [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+# Load data
+
+* First we'll load the sample info data.
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  load("../r_files/samples.Rda") 
+  load("../r_files/data_1.Rda") 
+```
+
+* Next we'll load the pollutnant data we're interested in. In this case we're going to look at the VOCs:
```
Comment (ngood13) [inline r_markdown/background_analysis_ecoc.Rmd:32]: VOCs?

File: r_markdown/background_analysis_ecoc.Rmd [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    theme_minimal() + ggtitle("OC rate")
+```
+
+## Plot background ecoc levels by date
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}  
+  ggplot(ec_bg, aes(date, ec_rate, color = qc)) +
+    geom_point() +
+    theme_minimal()
+  ggplot(oc_bg, aes(date, oc_rate, color = qc.x)) +
```
Comment (ngood13) [inline r_markdown/background_analysis_ecoc.Rmd:109]: is the join working as required? the .x and .y suffixes normally appear where a join have gone awry?

**Review by ngood13 — COMMENTED** [review]
Umm, I'm still worried that lots of these files that are changing aren't related to the smps background. I'll have to take a detailed look at everything tomorrow...

**Comment by kbilsback** [comment]
Grrrr I tried creating a new branch and then just changing the background and smps file...let me try again 

Kelsey Bilsback

File: r_markdown/background_analysis_ecoc.Rmd (hunk 2 above)
Comment (kbilsback) [reply r_markdown/background_analysis_ecoc.Rmd:109]: I think it usually just adds .x .y if you are joining and columns have the same name but i can fix

**Comment by kbilsback** [comment]
Wow, I did not realize when you replied to an email it got posted on github...

File: r_markdown/background_analysis_ecoc.Rmd (hunk 1 above)
Comment (kbilsback) [reply r_markdown/background_analysis_ecoc.Rmd:32]: copy/paste issue
